curr_scraper/settings.py

```diff
@@ -80,8 +82,13 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': os.environ.get('DB_NAME'),
+        'USER': os.environ.get('DB_USER'),
+        'PASSWORD': os.environ.get('DB_PASSWORD'),
+        'HOST': os.environ.get('DB_HOST'),
+        'PORT': os.environ.get('DB_PORT'),
```

Comment:
Please use the database URL for DB credentials as it's universal for most platforms/services that are usually used alongside Django.

Additionally, configure ATOMIC_REQUESTS setting for database.

BTW, feel free to use third party packages.

Reply:
Updated.

Reply:
@Fazzer6132 Please check this article on database url: https://stackoverflow.com/questions/3582552/what-is-the-format-for-the-postgresql-connection-string-url